### PR TITLE
Add support for `composer`.

### PR DESCRIPTION
**Pull request by senki**
A sincere attempt to basic support for composer global packages.

Note: I'm not an avid Python developer, instead I'm just interested to support composer as well.

### CODE REVIEW

File: meta_package_manager/managers/composer.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+        output = self.run([self.cli_path] + self.cli_args + [
+            'search', query])
+
+        if output:
+            regexp = re.compile(r'(\S+\/\S+) .*')
+            for package in output.split('\n'):
+                match = regexp.match(package)
+                if match:
+                    package_id = match[1]
```
Comment (kdeldycke) [inline meta_package_manager/managers/composer.py]: You should fetch the group here:
```suggestion
                    package_id = match.group(1)
```

File: meta_package_manager/managers/composer.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                    matches[package_id] = {
+                        'id': package_id,
+                        'name': package_id,
+                        'latest_version': 'N/A in composer',
```
Comment (kdeldycke) [inline meta_package_manager/managers/composer.py]: If you ca't fetch the latest version, just set it to `None`, as in https://github.com/kdeldycke/meta-package-manager/blob/65d51e439921cc397b16befe318c1ff00ccb83ce/meta_package_manager/managers/mas.py#L114:
```suggestion
                        'latest_version': None,
```

**Comment by kdeldycke** [comment]
Thanks @senki for the contribution! You're not far from having the basic tests pass. I added a couple of comments. If you apply these code changes, there's a change for your PR to be ready to be merged! :)

**Comment by kdeldycke** [comment]
Don't worry @senki, I'll fix the code myself. Thanks for the contribution.

**Comment by senki** [comment]
Hi @kdeldycke,

That was fast :)

Thank you for squashing bugs in my code.

**Pull request merged**